src/cijobs/project.json

```diff
@@ -15,11 +15,13 @@
     "netcoreapp1.0": {
         "dependencies": {
             "Microsoft.NETCore.App": {
-                "type": "platform",
                 "version": "1.0.0"
             }
         },
         "imports": "dnxcore50"
     }
+  },
+  "runtimes": {
```

Comment:
Note that cijobs has to run all all targets (mac, flavors of linux, etc. so turning this back into a standalone means you need to add lots more to the runtime section)